microsoft-edge/devtools-guide-chromium/customize/index.md

```diff
@@ -117,7 +117,7 @@ The next time you open DevTools, a new page named **Experiments** is displayed i
 <!-- links -->  
 
 [DevToolsPlacement]: ./placement.md "Change Microsoft Edge DevTools placement | Microsoft Docs"  
-[DarkTheme]: ./dark-theme.md "Enable Dark Theme in Microsoft Edge DevTools | Microsoft Docs"  
+[Theme]: ./theme.md "Enable Dark Theme in Microsoft Edge DevTools | Microsoft Docs"  
```

Comment:
Does this title (in quotes) need to be updated?

Reply:
Thanks for catching this. Yes, will update it.